### DISCUSSION TITLE
LunarVim git <leader> submenu

### DISCUSSION THREAD

**Discussion by pergatore** [LunarVim/LunarVim#3948] [Q&A] [answered]
Just a question from a still learning newbie; in the default config under <leader> -> g -> b you can only check out already created branches, is there a way there to check out a new branch that I'm not seeing? essentially `git checkout -b <name>`

If not, can one edit the default Leader submenus from the config.lua? With perhaps a:
```lua
lvim.builtin.which_key.mappings["g"] = {
   name = "+Git",
   B = { "FancyCommandGoesHere", "Create branch" },
},
```
Or would that overrite the current +Git menu?

**Comment by LostInTheLogs** [comment, marked as answer]
```lua
lvim.builtin.which_key.mappings.g.B = { "FancyCommandGoesHere", "Create branch" } -- this will only set the `B` key of the `g` table
lvim.builtin.which_key.mappings.g = {...} -- this overwrites the whole `g` table
```
you might want to learn more about lua tables
